vendor/assets/javascripts/codemirror-modes/gobstones.js

```diff
@@ -1,1059 +1,71 @@
-// Parts from Ace; see <https://raw.githubusercontent.com/ajaxorg/ace/master/LICENSE>
-CodeMirror.defineMode("gobstones", function(cmCfg, modeCfg) {
-
-  // Fake define() function.
-  var moduleHolder = Object.create(null);
-
-  // Given a module path as a string, create the canonical version
-  // (no leading ./, no ending .js).
-  var canonicalPath = function(path) {
-    return path.replace(/\.\//, '').replace(/\.js$/, '');
+// https://codemirror.net/demo/simplemode.html
+
+(function() {
+  var locale = document.querySelector("html").lang || 'es';
```

Comment:
This is not the same as previously removed line, since you are not using the generic Mumuki locale.

Reply:
Actually yes, it's the mumuki locale (https://github.com/mumuki/mumuki-laboratory/blob/20dcda48a752bbad5cdd4d10bf9fa6c5926b0eba/app/views/layouts/_main.html.erb#L2). I used it because I need the organization's locale code in a static way (the application's JS loads after the codemirror modes)

Reply:
OK, that means that there is no good reason to have a default value, there, since the language is always set, right?

Reply:
Anyway, I was going to suggest removing that default, but it looks to be already be in original code, so it is not actually part of this PR. Dismiss my comment.